testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/deployment/ironjacamar.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ironjacamar>
+    <connection-definitions>
+        <connection-definition
+                class-name="org.jboss.as.test.integration.jca.rar.MultipleManagedConnectionFactory1"
+                jndi-name="java:jboss/name1" pool-name="ij_Pool1">
+        </connection-definition>
+    </connection-definitions>
```

Comment:
This connection definition seems it is unused on your test, is so, you can remove it

Reply:
IMO there should be no relationship between the 2 test cases, because the deployment will be undeployed anyway before starting the next one, right? And updating the jndi bindings to something else does not work either.

Reply:
> because the deployment will be undeployed anyway before starting the next one, right? 

Yes it is undeployed and that was my initial thought but even upon an undeployment `IronJacamarDeploymentTestCase` fails after being affected by the deployment of `ironjacamar.xml` on `JndiViewOperationJMSTestCase`. I haven't got into the details, but seems it is due to the admin-objects defined there.

Reply:
@yersan yes, it is failing because of admin object.

Reply:
@parsharma For now, please use resource adapter's CLI like: 
```
/subsystem=resource-adapters/resource-adapter=eis.rar:add(archive=eis.rar)
/subsystem=resource-adapters/resource-adapter=eis.rar/admin-objects=ij_Pool:add(jndi-name="java:jboss/Name3", class-name="org.jboss.as.test.integration.jca.rar.MultipleAdminObject1Impl")
/subsystem=resource-adapters/resource-adapter=eis.rar:activate()
```
to activate the resource adapter instead of using `ironjacamar.xml`, there maybe a hidden bug in `resource-adapter` subsystem or `ironjacamar` when it is activted using `ironjacamar.xml` as @yersan pointed. :)

You can refer to [this test case](https://github.com/wildfly/wildfly/blob/master/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/ear/RarInsideEarReDeploymentTestCase.java#L74-L100) on how to do it. HTH

Reply:
@gaol  Thanks, your above suggestion works.